firmware/klipper_configurations/SKR_1.3/Voron2_SKR_13_Config.cfg

```diff
@@ -175,7 +177,8 @@ stealthchop_threshold: 0
 step_pin: z:P2.2
 dir_pin: !z:P2.6
 enable_pin: !z:P2.1
-step_distance: 0.00250
+rotation_distance: 8
```

Comment:
Can we try using gear_ratio for this? 👍 I think it would make a lot of sense, 16:80 (or maybe it's 80:16?) and then keep the 40mm rotation distance

Reply:
sure.  I'll have to test that out, and verify the correct settings for that.  I'll update the PR, well...once my current print finishes.

Reply:
And done